pkg/sbom/sbom.pb.go

```diff
@@ -101,11 +102,11 @@ func (x HashAlgorithm) String() string {
 }
 
 func (HashAlgorithm) Descriptor() protoreflect.EnumDescriptor {
-	return file_api_sbom_proto_enumTypes[0].Descriptor()
+	return file_sbom_proto_enumTypes[0].Descriptor()
```

Comment:
The protoc invocation is renaming these functions which is causing the diff that breaks the test. I think what is happening here is that for some reason, the protoc invocation that generated this file ran effectively from the `api/` dir and not from the repo root.

The same test should also cover the newly added gorm types.

Reply:
Fixed the invocation. Still need to rework the gorm options so that the other new content (e.g. the new `Document.Id` field accessor) isn’t written to the original sbom.pb.go. 

Since it’s adding a new file though, will the `git diff` check in the build test pass?

Reply:
(we discussed this on slack: the test passes and it covers the new generated code, thanks!)

Reply:
I didn't push this change, but limiting the `git diff` check in `verify-proto.sh` to just the compiled protobufs made it a little easier to test locally.

```shell
git diff --exit-code -- **/*.pb{,.gorm}.go || exit_with_msg ...
```